cmd/resim/commands/batch.go

```diff
@@ -625,6 +645,13 @@ func createBatch(ccmd *cobra.Command, args []string) {
 		body.PoolLabels = &poolLabels
 	}
 
+	// Sync metrics2.0 config
+	if viper.GetBool(batchSyncMetricsConfigKey) {
+		if err := SyncMetricsConfig(projectID, branchName, false); err != nil {
```

Comment:
nit: interface here should be `SyncMetricsConfig(projectID, branchID)` for consistency